components/date-time/date.js

```diff
@@ -2,7 +2,11 @@
  * External dependencies
  */
 import ReactDatePicker from 'react-datepicker';
-import moment from 'moment';
+
+/**
+ * WordPress dependencies
+ */
+import { moment } from '@wordpress/date';
```

Comment:
I'm confused about the utility of importing this package from `@wordpress/date` when it's already available as a regular dependency. It seems like more stuff we have to manage to export the same API (and if we aren't exposing `moment`'s exact API we should maybe call it something different). Why do this?

Reply:
I agree with this, the idea is that it absorbs the WP locale and timezone settings. But the downside is that this forces us somehow to stay compatible with the current moment version. We should definitely try to remove `moment` from the `date` module entirely, but let's leave it for a separate PR.